### PR TITLE
Redoing PR #383

### PR DESCRIPTION
**Pull request by j3mdamas**
Reference: #383

### CODE REVIEW

File: htmd/__init__.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
 from htmd.ui import *  # TODO: This will be deprecated in a few versions
-from htmd.config import config
```
Comment (stefdoerr) [inline htmd/__init__.py]: why are you removing config from __init__? It's actually necessary there. cannot be moved to ui

File: htmd/__init__.py [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 from htmd.version import version as _version
+from htmdx.cli import check_registration, show_news
+from htmd.latest import compareVersions
 import os
 
 config()
 __version__ = _version()
 
+if not (os.getenv("HTMD_NONINTERACTIVE")):
```
Comment (stefdoerr) [inline htmd/__init__.py]: Also registration check should be in ui to avoid unnecessary slowdown and messages when importing directly

**Comment by stefdoerr** [comment]
If you undo the changes to __init__ and ui I'll accept it :)

File: htmd/__init__.py (hunk 1 above)
Comment (j3mdamas) [inline htmd/__init__.py]: This was a mistake, already fixed it

**Pull request merged**